dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05355AddAppsPortletToLayout.java

```diff
@@ -0,0 +1,113 @@
+package com.dotmarketing.startup.runonce;
+
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.business.FactoryLocator;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+
+/**
+ * This startup task will update in the cms_layouts_portlets
+ * the portlet ids withthe new values easier to read
+ * 
+ * @author oswaldogallango
+ *
+ */
+public class Task03735UpdatePortletsIds implements StartupTask {
+
+	private Map<String, String> portletIds = new HashMap<String,String>();
```

Comment:
Codacy found an issue: [Private field 'portletIds' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5919676)